apostrophecms/form
# Feature/stepper

## Summary

Add group widget

## What are the specific steps to test this change?

1. Create a form
2. Add a group widget, it should contains all the other field widgets

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
